website/content/en/docs/config/environment-variables.md

```diff
@@ -57,6 +57,45 @@ This page documents the environment variables used in Lima.
   lima
   ```
 
+### `LIMA_SHELLENV_ALLOW`
+
+- **Description**: Specifies a comma-separated list of environment variable patterns to allow when propagating environment variables to the Lima instance with `--preserve-env`. When set, **only** variables matching these patterns will be passed through, completely overriding the default block list behavior.
```

Comment:
It should be also mentioned that this only applies to Lima v2.0 or later

Reply:
Isn't this is more general issue that we have one set of docs that are supposed to document the current and the upcoming release simultaneously? It would be better to have 2 separate versions, so the docs can be simpler.